src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -1817,7 +1817,11 @@ private void Log(string message)
 
         private void Window_PreviewMouseLeftButtonUp(object sender, MouseButtonEventArgs e)
         {
-            HidePopupWhenWindowDeactivated();
+            //if original sender was scroll bar(i.e Thumb) don't close the popup.
+            if(!(e.OriginalSource is Thumb))
+            {
+                HidePopupWhenWindowDeactivated();
```

Comment:
https://github.com/DynamoDS/Dynamo/commit/b0700d5e00023e3f883fda5e4869e93ba813ae98
This is the original commit where this code was added. I think, this fixed a bug where the incanvas search was open when you open another view (customnode) or another workspace. 
this check should fix that inappropriate close when mouse is released in the scroll.

Reply:
Hey @ramramps I just pulled the branch and tested locally.  The in-canvas search looks good and no longer collapses when scrolling.  I am not sure how critical it is but the search window remains open now when switching windows and appears in front of any other content you have open which was I think what you were describing above.  Such as 
![image](https://user-images.githubusercontent.com/13341935/37666639-6a151e5c-2c36-11e8-8aa5-e3d93850eb7f.png)

Reply:
Thanks @alfarok I will check this

Reply:
I was able to reproduce this on the master branch. This fix doesn't necessarily change that.